### PR TITLE
feat: add state attributes to deposit/redeem/compound events

### PR DESCRIPTION
**Pull request by apollo-sturdy**

### CODE REVIEW

File: contracts/locked-astroport-vault/src/execute/basic_vault.rs [hunk 1]
```diff
@@ -55,10 +59,13 @@ pub fn execute_deposit(
     }
     .into();
 
+    let state = STATE.load(deps.storage)?;
```
Comment (pacmanifold) [inline contracts/locked-astroport-vault/src/execute/basic_vault.rs:62]: why not load state at beginning of function instead of using mut deps?

**Review by pacmanifold — APPROVED** [review]
one comment otherwise lgtm

File: contracts/locked-astroport-vault/src/execute/basic_vault.rs (hunk 1 above)
Comment (apollo-sturdy) [reply contracts/locked-astroport-vault/src/execute/basic_vault.rs:62]: Because we want to load the state after the state has been updated, not before.

**Review by pacmanifold — APPROVED** [review]
lgtm

**Review by pacmanifold — APPROVED** [review]

**Pull request merged**